exporter/exporterhelper/queued_retry.go

```diff
@@ -110,6 +127,10 @@ func (qrs *queuedRetrySender) send(req request) (int, error) {
 		return qrs.consumerSender.send(req)
 	}
 
+	// We need to decouple the context we store for async requests from the
+	// context received by the receiver. The grpc/http based receivers will cancel
```

Comment:
"Decouple" does not fully explain the problem. Can you expand a bit? Perhaps say "prevent cancellation and deadline signals to propagate"?

Reply:
Done.